testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -57,6 +57,7 @@ pub struct EmitJob {
     workers: Vec<Worker>,
     stop: Arc<AtomicBool>,
     stats: Arc<StatsAccumulator>,
+    latencies: Arc<AtomicHistogram>,
```

Comment:
Would it make more sense to have `latencies: AtomicHistogram` as a field on `StatsAccumulator`?